src/lib/common/sol-mainloop.c

```diff
@@ -104,6 +105,12 @@ sol_init(void)
 
     sol_log_domain_init_level(SOL_LOG_DOMAIN);
 
+#ifdef MODULES
```

Comment:
check `SOL_DYNAMIC_MODULES` to keep it consistent (eventually change it in other places as well).